modules/ca/src/tools/caget.c

```diff
@@ -434,11 +436,16 @@ int main (int argc, char *argv[])
             enumAsNr = 1;
             break;
         case 'w':               /* Set CA timeout value */
+            /*
+             * epicsScanDouble is a macro defined as epicsParseDouble,
+             * (found in modules/libcom/src/misc) which will only
+             * change caTimeout here if it finds an acceptable value.
+             */
```

Comment:
Move the "only change the value if it converts successfully" into the epicsStdlib.h header. Ditto below.